app/components/Activity/TransactionModal/TransactionModal.js

```diff
@@ -72,38 +72,53 @@ export default class TransactionModal extends React.PureComponent {
             }
           />
 
-          <Bar />
+          {item.num_confirmations > 0 && (
+            <>
+              <Bar />
 
-          <DataRow
-            left={<FormattedMessage {...messages.date_confirmed} />}
-            right={
-              <>
-                <FormattedDate
-                  value={item.time_stamp * 1000}
-                  year="numeric"
-                  month="long"
-                  day="2-digit"
-                />
-                <br />
-                <FormattedTime value={item.time_stamp * 1000} />
-              </>
-            }
-          />
+              <DataRow
+                left={<FormattedMessage {...messages.date_confirmed} />}
+                right={
+                  item.num_confirmations ? (
+                    <>
+                      <FormattedDate
+                        value={item.time_stamp * 1000}
+                        year="numeric"
+                        month="long"
+                        day="2-digit"
+                      />
+                      <br />
```

Comment:
Is there a need for `<br/>` here ?

Reply:
Yes - I want this to show over 2 lines

Reply:
I mean it would probably be cleaner to do this with regular markup means. `brs` are usually not very flexible in terms of padding/margin etc

Reply:
Ahh, yeah sorry - don't know why that didn't click. I've updated to use `Text` components.